base_layer/wallet_ffi/src/lib.rs

```diff
@@ -2565,6 +2566,7 @@ pub unsafe extern "C" fn transport_type_destroy(transport: *mut TariTransportTyp
 /// `database_path` - The database path char array pointer which. This is the folder path where the
 /// database files will be created and the application has write access to
 /// `discovery_timeout_in_secs`: specify how long the Discovery Timeout for the wallet is.
+/// `network`: name of network to connect to.
```

Comment:
Better add a list of valid network strings here and in the header, a user of the client wouldn't know where to find that in the rust repo.